### ISSUE TITLE
Untrusted servers

### ISSUE THREAD

**Issue by Tentacular261** [hacki11/cisco-anyconnect-cli#6]
I am not able to connect to my work VPN server using this, because it tries to use my username to answer the question as to whether the untrusted connection setting should be change:

```
[INFO ] Using /opt/cisco/anyconnect/bin/vpn
[INFO ] Connecting to '<SERVER ADDRESS>' as '<USERNAME>'
[DEBUG] Cisco AnyConnect Secure Mobility Client (version 4.8.01090) .

Copyright (c) 2004 - 2019 Cisco Systems, Inc.  All Rights Reserved.


  >> state: Disconnected
  >> state: Disconnected

  >> Network error. Unable to lookup host names.
  >> Limited Access - DNS Failure
  >> registered with local VPN subsystem.
  >> contacting host (<SERVER ADDRESS>) for login information...
  >> notice: Contacting <SERVER ADDRESS>.

AnyConnect cannot verify server: <SERVER ADDRESS>
Connecting to this server may result in a severe security compromise!
AnyConnect is configured to block untrusted servers by default.  Most users choose to keep this setting.
If this setting is changed, AnyConnect will no longer automatically block connections to potentially malicious network devices.

Change the setting that blocks untrusted connections? [y/n]: <USERNAME>
  >> warning: Connection attempt has failed.
  >> state: Disconnected

```

**Comment by davidhcefx** [comment]
Same here!
Works fine when done manually, but failed when using this tool:
```sh
$ echo -e "y\n${USER}\n${PASSWORD}" | "/path/to/Cisco AnyConnect Secure Mobility Client/vpncli.exe" connect ${URL} -s
Cisco AnyConnect Secure Mobility Client (version 4.8.01090) .

Copyright (c) 2004 - 2019 Cisco Systems, Inc.  All Rights Reserved.


  >> state: Disconnected
  >> state: Disconnected
  >> notice: Ready to connect.
  >> registered with local VPN subsystem.
  >> contacting host (<SERVER>) for login information...
  >> notice: Contacting <SERVER>.
AnyConnect cannot verify server: <SERVER>
    - Certificate does not match the server name.
    - Certificate is from an untrusted source.
    - Certificate has an invalid date.
Connecting to this server may result in a severe security compromise!

Most users do not connect to untrusted servers unless the reason for the error condition is known.

Connect Anyway? [y/n]:
  >> Please enter your username and password.
.....
  >> notice: Establishing VPN...
  >> state: Connected
```

@hacki11, Looks like line 20 is the culprit:

https://github.com/hacki11/cisco-anyconnect-cli/blob/9139ad4af9d37c510ba004836b9b6e15dfc256ce/cisco_anyconnect_cli/cisco_anyconnect.py#L15-L22

**Comment by hacki11** [comment]
fixed with #9
